nautobot/core/tests/integration/test_navbar.py

```diff
@@ -0,0 +1,185 @@
+from django.test.utils import override_settings
+from selenium.common.exceptions import NoSuchElementException
+
+from nautobot.utilities.testing.integration import SeleniumTestCase
+from nautobot.utilities.choices import ButtonActionColorChoices, ButtonActionIconChoices
+
+
+class NavBarTestCase(SeleniumTestCase):
+    """Integration test the naviagation menu."""
+
+    fixtures = ["user-data.json"]
+    navbar = {
+        "Organization": {
+            "Sites": {
+                "Sites": {
+                    "permission": "dcim.view_site",
+                    "buttons": ["Add", "Import"],
+                },
+                "Regions": {
+                    "permission": "dcim.view_region",
+                    "buttons": ["Add", "Import"],
+                },
+            },
+            "Tags": {
+                "Tags": {
+                    "permission": "extras.view_tag",
+                    "buttons": ["Add", "Import"],
+                },
+            },
+        },
+        "Extensibility": {
+            "Data Management": {
+                "Relationships": {
+                    "permission": "extras.view_relationship",
+                    "buttons": [
+                        "Add",
+                    ],
+                },
+            },
+        },
+    }
+
+    def setUp(self):
+        super().setUp()
+        self.login(self.user.username, self.password)
+
+    def tearDown(self):
+        self.logout()
+        super().tearDown()
+
+    @override_settings(HIDE_RESTRICTED_UI=False)
+    def test_navbar_render_superuser(self):
+        """
+        Render navbar from home page with superuser.
+        """
+        # Set test user to admin
+        self.user.is_superuser = True
+        self.user.save()
+
+        # Retrieve home page
+        self.selenium.get(f"{self.live_server_url}")
+        self.selenium.wait_for_html("body")
```

Comment:
These two lines are repeated in several tests. Perhaps we should factor this into a helper-method on the `SeleniumTestCase`? Like `load_page(url)` or something?